### PR TITLE
feat: remove blur on click

### PR DESCRIPTION
**Pull request by hitendreamworld**

### CODE REVIEW

File: dw-icon-button.js [hunk 1]
```diff
@@ -206,44 +245,81 @@ export class DwIconButton extends buttonFocus(LitElement) {
...
+              </dw-tooltip>
+            `
+          : ""
+      }
+    `;
+  }
+
+  _onTooltipContainerClick(e) {
+    if (this.disabled) {
+      e.stopImmediatePropagation();
```
Comment (ChiragMoradiya) [inline dw-icon-button.js:317]: @hitendreamworld what does this do?

File: dw-icon-button.js [hunk 2]
```diff
@@ -252,56 +328,61 @@ export class DwIconButton extends buttonFocus(LitElement) {
    * @protected
    */
   _buttonStyle() {
-    if(!this.buttonSize) {
-      return '';
+    if (!this.buttonSize) {
```
Comment (ChiragMoradiya) [inline dw-icon-button.js:331]: There are lots of formatting changes included in this PR. It makes review process difficult.

Standard Practices:
1. Whenever you find something isn't properly formatted; please submit a separate PR for formatting, OR Just commit them into `main` or `master` branch with proper message & `[ci skip]` tag. 
2. It seems you are using a VS Code Plugin, which does this auto-formatting. So, let's define some common tools to be used by Team, and document them somewhere, so our tool-set can be uniform across team.

File: dw-icon-button.js [hunk 3]
```diff
@@ -185,18 +198,44 @@ export class DwIconButton extends buttonFocus(LitElement) {
        */
       title: { type: String },
 
+      /**
+       * Input property.
+       * Disabled Tooltip text.
+       */
+      disabledTitle: { type: String },
```
Comment (ChiragMoradiya) [inline dw-icon-button.js:205]: It seems you introduced this as a new feature. But, the card's title or scope doesn't mention about it.

File: dw-icon-button.js [hunk 4]
```diff
@@ -185,18 +198,44 @@ export class DwIconButton extends buttonFocus(LitElement) {
...
        * Possible values: FILLED and OUTLINED
        */
-      iconFont: { type: String, reflect: true }, 
+      iconFont: { type: String, reflect: true },
+
+      /**
+       * When it's `true`, shows `Material Symbols` icon.
+       * [Reference](https://fonts.google.com/icons?icon.set=Material+Symbols)
+       */
+      symbol: { type: Boolean, reflect: true },
```
Comment (ChiragMoradiya) [inline dw-icon-button.js:218]: This also seems a new feature. What was the previous way to show icon?

File: dw-icon-button.js [hunk 5]
```diff
@@ -206,44 +245,81 @@ export class DwIconButton extends buttonFocus(LitElement) {
...
-      <dw-tooltip
-        .trigger=${"mouseenter"}
-        .forEl=${this}
-        .offset=${[0, 8]}
-        .extraOptions=${{ delay: [500, 0] }}
-        .content=${this.title}>
-      </dw-tooltip>
-      ` : ''}
-    `
+      <div id="wrapper" @click=${this._onTooltipContainerClick}>
```
Comment (ChiragMoradiya) [inline dw-icon-button.js:269]: @hitendreamworld I don't understand why this is changed?

1. Why do we need to add wrapper?
2. Why do we need to stop propagation of the event when disabled?

**Review by ChiragMoradiya — CHANGES_REQUESTED** [review]

File: dw-icon-button.js (hunk 4 above)
Comment (nirmalbaldaniya-dream) [reply dw-icon-button.js:218]: @ChiragMoradiya `Symbols` had been introduced by Material icons. So Whenever the user wants to use them, this property must be given. In `dw-icon` this feature was added already. So here `symbol` property is just proxied to it.

**Pull request merged**